src/DTO/Mapper/StatusMessageMapper.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\DTO\Mapper;
+
+use App\DTO\Model\StatusMessageDTO;
+use App\Entity\General\StatusMessage;
+
+class StatusMessageMapper
+{
+    public static function mapStatusBanner(StatusMessage $entity, string $lang): StatusMessageDTO
+    {
+        switch ($lang) {
+            case "it":
+                $message = $entity->getItMessage();
+                break;
+            case "fr":
+                $message = $entity->getFrMessage();
+                break;
+            case "de":
+            default:
+                $message = $entity->getDeMessage();
+        }
```

Comment:
maybe it would make sense to add a function `$entity->getMessage($lang)` and to move this switch case into there. What do you think of this?

If you think this makes more sense like it is implemented now you can leave it as is.